.github/workflows/ci-automated-pipeline.yaml

```diff
@@ -0,0 +1,155 @@
+name: '[CI/CD] CI Pipeline for automated PRs'
+on: # Triggered only on open PR events done by bitnami-bot on containers folders
+  pull_request_target:
+    types:
+      - opened
+    branches:
+      - main
+      - bitnami:main
+    paths:
```

Comment:
Should we use this filter for the non-automated CI? After all, we are also filtering it for the CD.

Reply:
Good question! The `verify` action is mandatory for all PR. If we add the filter and we change for example this file, we couldn't merge the PR because the verify action is not triggered. Automated PR are a little bit special because the scope of them is very focused on the realease of particular branch

Reply:
I agree with you. The required status for the `Verify` job is key and it's not worthwhile to explore a workaround for this.